### ISSUE TITLE
How to clean up all the cache image

### ISSUE THREAD

**Issue by luojinghui** [jayesbe/react-native-cacheable-image#32]
in code, How to clean up all the cache image?

**Comment by abelaska** [comment]
That's a good point. Function to remove all cached images would be very useful.

**Comment by jayesbe** [comment]
Agreed. There should be a mechanism to remove the cached file. I think a static function for removing all cached images would work. Something like CacheableImage.clearCache(); 

For individual components I am thinking a property like 'clearCacheOnDestroy' as a boolean. Once set, on componentWillUnmount() it should check to see if the prop is set and if so remove it from the cache.

Thoughts ?

**Comment by luojinghui** [comment]
So, now only manually to delete the cache, and can not function to delete these pictures?

**Comment by wachunei** [comment]
I was looking at the source code and it actually has a method that deletes the cached file (`_deleteFilePath`, [here it is in use](https://github.com/jayesbe/react-native-cacheable-image/blob/master/image.js#L82)).

This method argument is `filePath`, which is calculated [here at line 52](https://github.com/jayesbe/react-native-cacheable-image/blob/master/image.js#L52), and uses `cachePath` and `cacheKeys` as variables.

Both variables are assigned in [`_processSource` method](https://github.com/jayesbe/react-native-cacheable-image/blob/master/image.js#L125).

So what would I do in order to delete a specific image from the cache:
* Create another method to calculate `cacheKey` (since `cachePath` is essentially source's URI host). Of course use this method in `_processSource`.
* Create a static method, let's say: `clearCachedFile` which would take a string (the URI), calculate host and `cacheKey`, then creates `filePath` variable and finally call `_deleteFilePath` with this variable.

@jayesbe What do you think? If is there something I'm missing let me know.

If this is fine for you I'd happily create a PR with this.

**Comment by jayesbe** [comment]
@wachunei This makes sense to me. PR is definitely welcome.

**Comment by wachunei** [comment]
@jayesbe will look into this next days!
